docs/docs-content/getting-started/aws/scale-secure-cluster.md

```diff
@@ -0,0 +1,505 @@
+| **Instance Type**     | `m4.2xlarge`                       | Set the compute size equal to the already provisioned nodes.                                                                                                                                                                                                                                 |
+| **Availability Zone** | _Availability zone of your choice_ | Set the availability zone the same as the already provisioned nodes.                                                                                                                                                                                                                         |
+
+Click on **Confirm**. The dialog closes. Palette begins provisioning your node pool. Once the process completes, your
+three node pools appear in a healthy state.
+
+![New worker pool provisioned](/getting-started/aws/getting-started_scale-secure-cluster_third-node-pool.webp)
+
+Navigate back to your terminal and execute the following command in your terminal to view the nodes of your cluster.
+
+```shell
+kubectl get nodes
+```
+
+The output reveals three nodes, two for worker pools and one for the control plane. Make a note of the names of your
+worker nodes. In the example below, `ip-10-0-1-133.ec2.internal` and `ip-10-0-1-32.ec2.internal` are the worker nodes.
+
+```shell
+NAME                         STATUS   ROLES           AGE   VERSION
+ip-10-0-1-32.ec2.internal    Ready    <none>          16m   v1.28.11
+ip-10-0-1-133.ec2.internal   Ready    <none>          46m   v1.28.11
+ip-10-0-1-95.ec2.internal    Ready    control-plane   51m   v1.28.11
+```
+
+It is common to dedicate node pools to a particular type of workload. One way to specify this is through the use of
+Kubernetes taints and tolerations.
+
+Taints provide nodes with the ability to repel a set of pods, allowing you to mark nodes as unavailable for certain
+pods. Tolerations are applied to pods and allow the pods to schedule onto nodes with matching taints. Once configured,
+nodes do not accept any pods that do not tolerate the taints.
+
+The animation below provides a visual representation of how taints and tolerations can be used to specify which
+workloads execute on which nodes.
+
+![Taints repel pods to a new node](/getting-started/getting-started_scale-secure-cluster_taints-in-action.gif)
+
+Switch back to Palette in your web browser. Navigate to the left **Main Menu** and select **Profiles**. Select the
+cluster profile deployed to your cluster, named `aws-profile`. Ensure that the **1.1.0** version is selected.
+
+Click on the **hellouniverse 1.1.3** layer. The manifest editor appears. Set the
```

Comment:
We could add a screenshot here :)

Reply:
I didn't think a screenshot of an edited manifest would be that helpful. I have added a code snippet with the tolerations section of the manifest instead.